### DISCUSSION TITLE
Getting `Action error: Cannot find package '@typeschema/zod' imported from /var/task/.next/server/chunks/962.js` error when deploying next app to Vercel with next-safe-action

### DISCUSSION THREAD

**Discussion by nikivdev** [next-safe-action/next-safe-action#140] [General]
Not sure why but I am getting a crash and this error in Vercel logs:

```
Action error: Cannot find package '@typeschema/zod' imported from /var/task/.next/server/chunks/962.js
```

<img width="1461" alt="image" src="https://github.com/TheEdoRan/next-safe-action/assets/6391776/d6b06aea-bfdf-40ab-bc2b-10cec21139fd">

Code: https://github.com/kuskusapp/kuskus
Actions Code: https://github.com/kuskusapp/kuskus/blob/main/app/actions.ts

**Comment by nikivdev** [comment]
This solution by @TheEdoRan https://github.com/TheEdoRan/next-safe-action/issues/64#issuecomment-1987478504

did not solve it

still getting same error 😿

**Comment by nikivdev** [comment]
https://github.com/kuskusapp/kuskus/commit/36cbd81911ce2039a902429c9a82347443a4c82a#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0 attempt at fix

**Comment by TheEdoRan** [comment]
Saw you switched to zsa in the code you linked above so this shouldn't be a problem anymore. Anyways, I can't reproduce the issue: [repo](https://github.com/TheEdoRan/next-minimal-actions-test).
